source/general/wikicontributing.rst

```diff
@@ -7,61 +7,63 @@ Contributing to the Wiki
 Setting up the environment
 ==========================
 
-To begin editing the wiki we first need to set our environment up. The prerequisites needed is just `python <https://www.python.org/downloads/>`_.
+To begin editing the Wiki we first need to up our environment. `Python`_ is the only dependency required.
 
-.. note:: When installing python make sure too tick the ``Add Python to PATH`` option to be able to use python within your terminal/command prompt
+.. note:: When installing Python make sure too tick the ``Add Python to PATH`` option to be able to use python within your terminal/command prompt
 
-First thing you need to do is to fork the git repository and then clone it.
+
+1. Fork the GitHub repository and clone it.
 
 .. note:: Remember to replace *exampleusername* with your actual github username
```

Comment:
Maybe make this all caps to make it more obvious?

Reply:
I feel that if the user is planning on contributing, they hopefully know to read everything on here, especially if they attempt to make a PR with changes.

Reply:
Better safe than sorry, making it all caps makes it much more obvious.

Reply:
Change to `YOUR_USERNAME`